pkg/security/metrics/metrics.go

```diff
@@ -255,6 +255,15 @@ var (
 	// MetricSecurityProfileCacheMiss is the name of the metric used to report the count of Security Profile cache misses
 	// Tags: -
 	MetricSecurityProfileCacheMiss = newRuntimeMetric(".security_profile.cache.miss")
+	// MetricSecurityProfileAnomalyDetection
+	// Tags: - event_type
+	MetricSecurityProfileAnomalyDetectionSent = newRuntimeMetric(".security_profile.anomaly_detection.sent")
+	// MetricSecurityProfileEventFilteringLookup
+	// Tags: -
+	MetricSecurityProfileEventFilteringLookup = newRuntimeMetric(".security_profile.event_filtering.lookup")
```

Comment:
Unless I'm missing something, I think `security_profile.event_filtering.lookup` == SUM(`security_profile.event_filtering`) no ? So in other words, I think you can remove the `security_profile.event_filtering.lookup` metric because we have the same information with another one.

I would also rename the metric `security_profile.event_filtering` to `security_profile.evaluation.hit` and `present_in_profile: yes / no` to `in_profile: true / false`.